fieldservice/views/fsm_team.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Field Service Team -->
+    <record id="fsm_team_tree_view" model="ir.ui.view">
+        <field name="name">Field Service Team List</field>
+        <field name="model">fsm.team</field>
+        <field name="arch" type="xml">
+            <tree string="Field Service Teams">
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="fsm_team_form_view" model="ir.ui.view">
+        <field name="name">Field Service Team Form</field>
+        <field name="model">fsm.team</field>
+        <field name="arch" type="xml">
+            <form string="Field Service Team">
+                <sheet>
+                    <group>
+                        <field name="name"/>
+                        <field name="description"/>
+                        <field name="stage_ids"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_fsm_team" model="ir.actions.act_window">
+        <field name="name">teams</field>
+        <field name="res_model">fsm.team</field>
+        <field name="view_id" ref="fsm_team_tree_view"/>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Create a Field Service Team.
+            </p>
+            <p>
+                Each Team addresses adiffernt type of work, and ca have their specific workflows. 
```

Comment:
`Each team addresses different type of work, and can have their specific workflow.`